x-pack/plugins/canvas/server/routes/workpad.test.js

```diff
@@ -72,21 +72,13 @@ Object {
   "id": "123",
 }
 `);
-      expect(savedObjectsClient.get).toMatchInlineSnapshot(`
-[MockFunction] {
-  "calls": Array [
-    Array [
-      "canvas-workpad",
-      "123",
-    ],
+      expect(savedObjectsClient.get.mock.calls).toMatchInlineSnapshot(`
+Array [
+  Array [
+    "canvas-workpad",
+    "123",
   ],
-  "results": Array [
```

Comment:
it doesn't make sense to test `jest` internals, so I removed part of snapshot test (yes, it had a conflict with new jest version)